UnitTests/Tests/PrecipitationSpec.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import Quick
+import Nimble
+
+class PrecipitationSpec: QuickSpec {
+    override func spec() {
+        describe("Precipitation") {
+            context("precipitation chance") {
+                it("returns none for 0% chance of precipitation") {
+                    let precipitation = Precipitation(probability: 0.0, type: "")
+                    expect{precipitation.chance}.to(equal(PrecipitationChance.None))
```

Comment:
Oh, didn't see that. There is trailing closure syntax in this function so this is fine by using this but this was accident that I did it. We can leave it as is or change if you want.